source/macos/AdaptiveCards/AdaptiveCards/Renderers/TextInputRenderer.swift

```diff
@@ -69,3 +120,35 @@ class ACRTextInputView: NSTextField, NSTextFieldDelegate {
         }
     }
 }
+class ACRCustomTextInputView: NSView {
+    public var placeholder: String?
+    public var stringValue: String?
+    
+    private lazy var stackview: NSStackView = {
+        let view = NSStackView()
+        view.orientation = .horizontal
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Do we need this class? I don't see any use for this?
We can directly use a stackView instead right?